webrtc.html

```diff
@@ -9849,6 +9851,40 @@ <h2>Methods</h2>
         </section>
       </div>
     </section>
+    <section>
+      <h3>canInsertDTMF update</h3>
```

Comment:
This is all good, but was invokes the canInsertDTMF update? The [[CanInsertDTMF]] attribute is initially false and never touched as far as I can tell.

This needs to be invoked either on all possible places where one of the conditions might change which seems tedious and error prone, or any time where we care that the internal slot is up to date, that is:
- When the canInsertDTMF attribute is invoked.
- When insertDTMF() is invoked.

Reply:
Added a step to invoke the update when insertDTMF() is called.

Reply:
What about when canInsertDTMF attribute is called? The caller might decide never to call insertDTMF() in the first place if canInsertDTMF is false.

Reply:
Fixed.

Reply:
the relevant webrtc code goes

if (canInsertDtmf()) {
  insertDTMF()
} else {
  return error
}

so this makes the spec match the implementation....

Reply:
If you skip that test it still throw an error... 🤷🏻‍♂️